Templates/TodoList-NodeJS/index.js

```diff
@@ -0,0 +1,45 @@
+﻿module.exports = function (context, req, inputList) {
+    var method = req.method.toLowerCase();
+
+    if (methods[method]) {
+        methods[method](context, req, inputList);
+    } else {
+        context.res = { status: 404, body: "No route found for VERB: " + req.method };
+    }
+
+    context.done();
+};
+
+var methods = {
+    get: function (context, req, inputList) {
+        var itemList = [];
+
+        // Filtering the input list to only get Item and completed column
+        for (var i in inputList) {
+            itemList.push({
+                "Item": inputList[i].item, "Completed": inputList[i].completed
+            });
+        }
+
+        context.res = {
+            body: itemList
+        };
+    },
+    post: function (context, req) {
+        var statusCode = 400;
+        var responseBody = "Invalid request object";
+
+        if (typeof req.body != 'undefined' && typeof req.body == 'object') {
+            statusCode = 201;
+            context.bindings.outList = req.body;
+            responseBody = "List Item Created";
+        }
+
+        context.res = {
+            status: statusCode,
+            body: responseBody
+        };
+    }
+    
+    // Updating an existing entity is currently supported only through C#    
```

Comment:
I'd remove this comment line. When you say this is possible in C#, you mean by using the storage SDK directly? You can do that in Node as well.